code/game/machinery/doors/airlock.dm

```diff
@@ -896,10 +896,21 @@ About the new airlock wires panel:
 					spawn(0)	close(1)
 				else
 					user << "<span class='warning'>You need to be wielding \the [C] to do that.</span>"
+
+	else if(istype(C, /obj/item/weapon/melee/misc/hammer) && !arePowerSystemsOn())

```

Comment:
I gave it a bit of thought and talk on Discord and decided that giving the hammer the same abilities of the crowbar (like the incredibly irritating occurrence in combat where the crowbar is constantly prying floor tiles when you miss) would be a really bad idea.